pyproject.toml

```diff
@@ -0,0 +1,102 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+asyncio_mode = "auto"
+markers = ["unstable"]
+
+# Formatting tools configuration
+[tool.black]
+line-length = 99
+target-version = ["py310"]
+
+[tool.isort]
+profile = "black"
+
+[tool.poetry]
+name = "spark-integration-hub-k8s"
+version = "1.0"
+description = "spark-integration-hub-k8s"
+authors = []
+
+[tool.poetry.dependencies]
+python = ">3.8,<4.0"
+ops = ">=2.10.1"
+pyyaml = ">=6.0.1"
+boto3 = ">=1.34.55"
+jsonschema = "^4.19.1"
+lightkube = ">=0.15.2"
+spark8t = { git = "https://github.com/canonical/spark-k8s-toolkit-py.git", branch="dpe-3519"}
```

Comment:
I guess we can already pin this to 0.0.7 now.

Reply:
Moved to test dependencies